mappings/net/minecraft/world/timer/Timer.mapping

```diff
@@ -6,6 +6,9 @@ CLASS net/minecraft/class_236 net/minecraft/world/timer/Timer
 	FIELD field_1315 LOGGER Lorg/apache/logging/log4j/Logger;
 	METHOD <init> (Lnet/minecraft/class_233;)V
 		ARG 1 timerCallbackSerializer
+	METHOD method_22592 getEventId ()Ljava/util/Set;
```

Comment:
```suggestion
	METHOD method_22592 getEventIds ()Ljava/util/Set;
```

Reply:
+1

Reply:
@liach  patched and will be pushed in a couple seconds

Reply:
https://github.com/FabricMC/yarn/pull/1218/commits/37fd9ec85575e410c8ab0dd94c541d99292ee9e7